src/datastore/postgres/schema/v6.ts

```diff
@@ -0,0 +1,34 @@
+import { IDatabase } from "pg-promise";
+import { MatrixUser } from "matrix-appservice-bridge";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create database
+    await db.none(`
+        CREATE TABLE linked_accounts (
+            user_id TEXT NOT NULL,
+            slack_id TEXT NOT NULL,
+            team_id TEXT NOT NULL,
```

Comment:
don't you want this to be a foreign key to `teams.id`?

Reply:
We tend to avoid foreign keys, at the moment. It's technically valid to have a linked_account entry with no associated team entry. If we run into issues though, you can blame me :p